projects/storefrontlib/src/lib/my-account/models/order.model.ts

```diff
@@ -0,0 +1,5 @@
+export interface UserOrders {
```

Comment:
Why not use `OrderHistoryList` from `projects/core/src/occ-models/occ.models.ts`

Reply:
Because UserOrderState has keys `orders; pagination; **sort**;` and OrderHistoryList has keys `orders; pagination; **sorts**;`.

Reply:
If you would like me to Change the UserOrderState, and its reducer, and every file that uses it, I will. But, that seemed outside of the scope of this ticket.

Reply:
Yes please, it will greatly improve typesafety

Reply:
My-account is using `OrderHistoryList`.
`UserOrderState`'s `orders` is not (should not) be of type `Order[]`, because it contains `total` property.

Reply:
Check the swagger when not sure: https://dev-com-17.accdemo.b2c.ydev.hybris.com:9002/rest/v2/swagger-ui.html#!/Orders/getOrdersForUserUsingGET

Reply:
Wouldn't something have failed then? If I was using the wrong type

Reply:
I'm surprised it didn't. Can you please address it?